### ISSUE TITLE
Two "Fitting" menus (Trac #403)

### ISSUE THREAD

**Issue by ajj** [SasView/sasview#537]
On the Mac (OS X 10.10) two fitting menus appear. This might seem to just be cosmetic, but the menus do not confer the same functionality - for instance the Constrained/Simultaneous Fit option only becomes un-greyed in the "first" menu and not in the second.

In any case, this is very confusing for the user.

Migrated from http://trac.sasview.org/ticket/403
```json
{
    "status": "closed",
    "changetime": "2015-04-06T06:34:08",
    "_ts": "2015-04-06 06:34:08.265091+00:00",
    "description": "On the Mac (OS X 10.10) two fitting menus appear. This might seem to just be cosmetic, but the menus do not confer the same functionality - for instance the Constrained/Simultaneous Fit option only becomes un-greyed in the \"first\" menu and not in the second.\n\nIn any case, this is very confusing for the user.",
    "reporter": "ajj",
    "cc": "",
    "resolution": "duplicate",
    "workpackage": "SasView Bug Fixing",
    "time": "2015-04-05T18:27:31",
    "component": "SasView",
    "summary": "Two \"Fitting\" menus",
    "priority": "blocker",
    "keywords": "",
    "milestone": "SasView 3.1.0",
    "owner": "",
    "type": "defect"
}
```

**Comment by ajj** [comment]
Trac update at `2015/04/05 18:28:27`: **ajj** commented:

NB tested on build http://trac.sasview.org/ticket/143 from april 2nd

**Comment by butlerpd** [comment]
Trac update at `2015/04/06 06:34:08`: 
* **butler** commented:

This is effectively a dupliate of a previoius ticket #524.  The text of this ticket has been copied to that ticket and the priority significantly raised to match this ticket.  I will now close this ticket as a duplicate.  Future comments to this issue should be addressed to ticket #524.


* **butler** changed resolution from "" to "duplicate"
* **butler** changed status from "new" to "closed"
